src/store/AppInfo.ts

```diff
@@ -56,6 +57,7 @@ export default {
     faucetUrl: (state) => state.faucetUrl,
     defaultFee: (state) => state.defaultFee,
     defaultWallet: (state) => state.defaultWallet,
+    fetchingTransactions: (state) => state.fetchingTransactions,
```

Comment:
`isFetchingTransactions` seems to be more natural as indicating "boolean" type with `is` annotation